dashboard/app/views/admin_search/pilots.html.haml

```diff
@@ -1,4 +1,21 @@
 %h3 Pilot Experiments
-- Experiment::PILOT_EXPERIMENTS.each do |pilot|
+
+- @pilots.each do |pilot|
   %p
-    = link_to pilot[:label], show_pilot_path(pilot_name: pilot[:name])
+    = link_to pilot[:display_name], show_pilot_path(pilot_name: pilot[:name])
+
+%hr
+
+%h3 Add a Pilot
+
+= form_for Pilot.new, url: {controller: 'admin_search', action: 'create_pilot'} do |f|
+  .field
+    = f.label 'Pilot name (all lowercase, no spaces)'
```

Comment:
What happens if someone provides an invalid pilot name?

Reply:
They get back some json with an error message in it (which the browser will display).  It's not great, but I didn't see a consistent error-handling pattern in this part of the code and didn't want to invest too much time into it.